tests/test_cli.py

```diff
@@ -41,3 +48,33 @@ def test_run(script_info):
     res = runner.invoke(cli.run, ['-d', '-c', '2'], obj=script_info)
     assert res.exit_code == 0
     assert 'Starting 2 tasks' in res.output
+
+
+def test_reindex(app, script_info, queue):
+    """Test reindex."""
+    # load records
+    with app.test_request_context():
+        runner = CliRunner()
+
+        schema = {
+            'type': 'object',
+            'properties': {
+                'title': {'type': 'string'},
+                'field': {'type': 'boolean'},
+            },
+            'required': ['title'],
+        }
+        record = Record.create({'title': 'Test0', '$schema': schema})
+        recid_minter(record.model.id, record.model.json)
+        with db.session.begin_nested():
+            db.session.add(record.model)
+        rec_id = record.id
```

Comment:
Please use ``rec_uuid`` instead of ``rec_id`` as otherwise it's very easy to confuse with the Persistent Identifier recid (which is an integer) vs the record model identifier (which is a uuid)